aten/src/ATen/native/mps/operations/UnaryOps.mm

```diff
@@ -250,6 +251,93 @@ void unary_op(const Tensor& self,
   });
 }
 
+TORCH_IMPL_FUNC(erfinv_out_mps)(const Tensor& self, const Tensor& output) {
+  mps::unary_op(self, output, "erfinv_out_mps", ^MPSGraphTensor*(MPSGraph* mpsGraph, MPSGraphTensor* inputTensor) {
+    auto dataType = inputTensor.dataType;
```

Comment:
Can you please provide a link or in comment provide formula you are implementing for this op?